playbooks/installer/packstack/post.yml

```diff
@@ -0,0 +1,145 @@
+
+    - name: Create new endpoint
+      shell: source /root/keystonerc_admin; keystone endpoint-create
+                --publicurl http://{{ ansible_default_ipv4.address }}:3333
+                --internalurl http://{{ ansible_default_ipv4.address }}:3333
+                --adminurl http://{{ ansible_default_ipv4.address }}:3333
+                --service {{ endpoint_s3.stdout }}
+      when: old_endpoint.stdout and endpoint_s3.stdout
+      register: new_s3_endpoint
+
+    - name: Delete old endpoint
+      shell: source /root/keystonerc_admin; keystone endpoint-delete {{ old_endpoint.stdout }}
+      when: new_s3_endpoint.changed
+
+    - name: Install openstack-nova-objectstore package
+      yum: name=openstack-nova-objectstore state=present
+
+    - name: Start and enable openstack-nova-objectstore
+      service: name=openstack-nova-objectstore state=started enabled=yes
+
+    - name: Open firewall for objectstore api
+      shell: |
+          ( iptables -C INPUT -p tcp -m multiport --dports 3333 -m comment --comment "bz1138740 - nova objectstore api incoming" -j ACCEPT && echo "Already open" ) || \
+          ( iptables -A INPUT -p tcp -m multiport --dports 3333 -m comment --comment "bz1138740 - nova objectstore api incoming" -j ACCEPT && echo "Opened" )
+
+- name: Check if BZ1210411 packstack/floating_ip_pool is enabled
+  hosts: controller
+  gather_facts: no
+  sudo: no
+  tasks:
+      - group_by: key=workaround_bz1210411
+        when: workarounds.bz1210411 is defined
+
+- name: "Workaround BZ1210411: Update floating_ip_pool in nova.conf"
+  hosts: workaround_bz1210411
+  gather_facts: yes
+  sudo: yes
+  tasks:
+    - name: Get packstack's floating ip pool value
+      shell: crudini --get packstack_config.txt general CONFIG_NOVA_NETWORK_DEFAULTFLOATINGPOOL
+      args:
+        chdir: /root
+      register: packstack_floating_ip_pool
+
+    - name: Apply packstack's floating ip pool value to nova.conf
+      shell: crudini --set /etc/nova/nova.conf DEFAULT default_floating_pool {{ packstack_floating_ip_pool.stdout }}
+
+    - name: Restart nova services to apply changes in nova.conf
+      shell: openstack-service restart nova
+
+    - name: Restart neutron service to apply changes in floating ip pool
+      shell: openstack-service restart neutron
+
+- name:  Post install for Neutron server
+  hosts: controller
+  gather_facts: no
+  sudo: yes
+  tasks:
+    - name: Enable l3-ha (VRRP) in neutron
+      ini_file:
+        dest: /etc/neutron/neutron.conf
+        section: DEFAULT
+        option: l3_ha
+        value: true
+      when: installer.network.ha is defined and installer.network.ha
+      register: ha
+
+    - name: Enable port-security in neutron
+      ini_file:
+        dest: /etc/neutron/plugins/ml2/ml2_conf.ini
+        section: ml2
+        option: extension_drivers
+        value: port_security
+      when: installer.network.type == 'neutron' and installer.product.version in ['7', '8']
```

Comment:
Why not to check if the installer.product.version >= 7? (just like in the original playbook)
Does't this suppose to run for product versions greater than 8?